### DISCUSSION TITLE
Many questions about KoboldAI

### DISCUSSION THREAD

**Discussion by kbuwel** [KoboldAI/KoboldAI-Client#405] [Q&A]
1. KoboldAI artificial intelligence changes situations randomly. 
I arbitrarily changed my age to say I was 10 years old and now I am a college student. 
The situation is like this. 
I created a situation where my family and friends were around age and gender. 
He said he was looking in the mirror, but AI suddenly changed the situation to say he had an 
argument with his dad and went back to college. 
KoboldAI_GPT-Neo-2.7B-AID 
KoboldAI_GPT-Neo-2.7B-Shinen 
I only received two models and ran them with KoboldAI_GPT-Neo-2.7B-AID. 
How do I avoid randomly changing situations? 
2. How do I get AIDungeon World information? 
It says to open a .json file, but I don't have a .json file. 
Where do you get it? 
README.md does not contain this information. 
I want to do AIDungeon's classic model, but is there any way? 
3. I ran aetherroom.club Prompt, but AI does not work. What is the reason? 
I write diligently, but in KoboldAI it shows Execution time: 0 sec. 
And the AI doesn't work. 
4. After executing remote-play.bat, local disk b was created. 
I would like to know how to remove it.

**Comment by henk717** [comment]
1. You are using old small models, so this is normal. Best way to avoid is to use something newer and larger such as the new Tiefighter model we released this week. You will however need to use our development version KoboldAI United to be able to use that model or switch to https://koboldai.org/cpp (The last one needs GGUF versions of the models you can obtain from huggingface.co).
2. AI Dungeon importing is no longer possible, all the conversion tools that were used to do so were made for the AI Dungeon website in 2020 and broken. We still allow the import in the menu but I am not aware of any tool that still can download it in those formats.
3. You would have to check the KoboldAI console for that to see if any error happened.
4. There is a disconnect-b-drive.bat on the B: drive, if you don't have it restarting (This is not the same as shutting down and starting) the PC will also get rid of it.
